src/core/modules/StateEvaluator.ts

```diff
@@ -236,6 +238,8 @@ export interface EvaluationOptions {
   // whether a separate key-value storage should be used for the contract
   useKVStorage: boolean;
 
+  useConstructor: boolean;
```

Comment:
can you please add a comment above the new parameter? explaining what exactly given parameter means.

Reply:
Sure I have done this, someone must have eaten it :)
Added :)